.gitignore

```diff
@@ -102,3 +102,18 @@ dist
 
 # TernJS port file
 .tern-port
+
+# Keys
+/neardev
+
+# Testing
+/coverage
+
+# Misc
+.DS_Store
```

Comment:
BUtttttttttttttttttt i like these files in git...

Reply:
:angry: :imp: